### PR TITLE
curlie: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
@kbdharun

File: pages/linux/curlie.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# curlie
+
+> Curlie is a frontend to curl that adds the ease of use of httpie.
+> More information: <https://github.com/rs/curlie>.
+
+- Send a GET simple GET request:
```
Comment (CleanMachine1) [inline pages/linux/curlie.md:6]:
```suggestion
- Send a GET request:
```

File: pages/linux/curlie.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# curlie
+
+> Curlie is a frontend to curl that adds the ease of use of httpie.
+> More information: <https://github.com/rs/curlie>.
+
+- Send a GET request:
+
+`curlie httpbin.org/get`
```
Comment (CleanMachine1) [inline pages/linux/curlie.md:8]: can we add our token syntax to these links please.

Comment (suyashbhawsar) [reply pages/linux/curlie.md:8]: Okay, you should also check `xh` page as I have referred it. Feel free to add the suggestions.

Comment (CleanMachine1) [inline pages/linux/curlie.md:8]:
```suggestion
`curlie {{httpbin.org/get}}`
```

File: pages/linux/curlie.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Send a POST request:
+
+`curlie post httpbin.org/post name=john age:=25`
```
Comment (CleanMachine1) [inline pages/linux/curlie.md:12]:
```suggestion
`curlie post {{httpbin.org/post}} {{name=john}} {{age:=25}}`
```

File: pages/linux/curlie.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Send a GET request with query parameters (e.g. `first_param=5&second_param=true`):
+
+`curlie get httpbin.org/get first_param==5 second_param==true`
```
Comment (CleanMachine1) [inline pages/linux/curlie.md:16]:
```suggestion
`curlie get {{httpbin.org/get}} {{first_param==5}} {{second_param==true}}`
```

File: pages/linux/curlie.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- Send a GET request with a custom header:
+
+`curlie get httpbin.org/get header-name:header-value`
```
Comment (CleanMachine1) [inline pages/linux/curlie.md:20]:
```suggestion
`curlie get {{httpbin.org/get}} {{header-name:header-value}}`
```

**Review by CleanMachine1 — COMMENTED** [review]
This is our token syntax, please learn how to use it since it will solve a lot of reviewing and let PRs be merged faster.

In these cases, we show that the link can be changed and other factors such as parameters.

**Comment by suyashbhawsar** [comment]
> This is our token syntax, please learn how to use it since it will solve a lot of reviewing and let PRs be merged faster.
> 
> In these cases, we show that the link can be changed and other factors such as parameters.
> 
> https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax

Okay, thanks!

**Review by kbdharun — APPROVED** [review]
LGTM

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**